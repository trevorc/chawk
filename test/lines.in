40	2
3	.141
-2	2
