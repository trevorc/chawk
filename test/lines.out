42
3.141
0
